### ISSUE TITLE
[1.3.1] "Kid" header key?

### ISSUE THREAD

**Issue by blanchonvincent** [FusionAuth/fusionauth-issues#68]
### Description 
One month ago, in the header of the token, we had a key `kid` in the headers.
Now we do not have it anymore. Is it related to a config in FusionAuth or is it a BC?

### Screenshots
<img width="600" alt="screen shot 2019-01-29 at 11 50 53 am" src="https://user-images.githubusercontent.com/1580512/51892842-8c871e80-23bc-11e9-91aa-14f21b11a0f8.png">

**Comment by robotdan** [comment]
@blanchonvincent Yes you're correct. For clarify, if anyone else runs into this issue: 

When you use the Login API directly without an Application Id, you are logging in "globally" - that is to say, you are not logging into an Application.  This means we will use the JWT signing configuration specified in the System Settings which is considered the global - or default signing configuration. 

In this case, because the signing configuration is not tied to a FusionAuth application, there will be no `kid` in the JWT header. 

If you send that JWT back to FusionAuth for validation, we will use the global JWT configuration to verify the signature because there is no `kid`.
